network/discovery/dv5_service.go

```diff
@@ -56,6 +57,14 @@ type DiscV5Service struct {
 	dv5Listener Listener
 	bootnodes   []*enode.Node
 
+	// withExtraFiltering specifies whether we want to additionally filter discovered peers,
+	// since operator node and boot node are using the same discovery code (`DiscV5Service`) we
+	// want to do this extra filtering for operator node only
+	withExtraFiltering bool
```

Comment:
bootnode doesn't pull out peers from discovery to filter them so this is not needed

Reply:
Can you clarify why it needs `newDiscV5Service` service then ?